apps/apihealth/views.py

```diff
@@ -1,10 +1,20 @@
+from covidx.apps.apihealth.serializer import ApiHealthSerializer
+from django.contrib.auth import get_user_model
```

Comment:
Codacy found an issue: [third party import "from django.contrib.auth import get_user_model" should be placed before "from covidx.apps.apihealth.serializer import ApiHealthSerializer"](https://app.codacy.com/gh/Xcov19/covidX/pullRequest?prid=6329042)